src/components/reusables/Filter/Filter.tsx

```diff
@@ -0,0 +1,150 @@
+import * as React from 'react';
+import { useState, useEffect, useId, type ReactNode } from 'react';
+import { Button } from '@digdir/design-system-react';
+import cn from 'classnames';
+import { XMarkIcon } from '@navikt/aksel-icons';
+
+import { arraysEqual } from '@/resources/utils';
+import { usePrevious } from '@/resources/hooks';
+
+import type { FilterOption } from './utils';
+import { OptionDisplay } from './OptionDisplay';
+import classes from './Filter.module.css';
+import { Floatover } from './Floatover';
+import { FilterButton } from './FilterButton';
+
+export interface FilterProps {
+  options: FilterOption[];
+  label: string;
+  applyButtonLabel: string;
+  resetButtonLabel: string;
+  searchable?: boolean;
+  icon?: ReactNode;
+  value?: string[];
+  onApply?: (value: string[]) => void;
+  modalView?: boolean;
+  closeButtonAriaLabel?: string;
+}
+
+export const Filter = ({
+  options,
+  label,
+  applyButtonLabel,
+  resetButtonLabel,
+  icon,
+  value,
+  searchable,
+  modalView = false,
+  closeButtonAriaLabel = 'Close ' + label,
```

Comment:
Yeah 😅
I mean, this is only the default text, so it really is up to the user of the component to set it in their chosen language, but I suppose I could fix it to the language files since this is only in our repo.